atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/SweepQueueReadWriteTest.java

```diff
@@ -77,22 +76,32 @@ public void setup() {
                 .persistToBytes();
     }
 
-    public static int writeTs(KvsSweepQueueWriter writer, long timestamp, boolean conservative) {
-        return writeTsToCell(writer, timestamp, Cell.create(ROW, COL), conservative);
+    public static int writeToDefault(KvsSweepQueueWriter writer, long timestamp, boolean conservative) {
+        return writeToCell(writer, timestamp, DEFAULT_CELL, conservative);
     }
 
-    public static int writeTsToCell(KvsSweepQueueWriter writer, long timestamp, Cell cell, boolean conservative) {
-        WriteInfo write = WriteInfo.of(conservative ? TABLE_REF : TABLE_REF2, cell, timestamp);
+    public static int writeToCell(KvsSweepQueueWriter writer, long timestamp, Cell cell, boolean conservative) {
+        return write(writer, timestamp, cell, false, conservative);
+    }
+
+    public static int putTombstone(KvsSweepQueueWriter writer, long timestamp, Cell cell, boolean conservative) {
+        return write(writer, timestamp, cell, true, conservative);
+    }
+
+    private static int write(KvsSweepQueueWriter writer,
+            long timestamp, Cell cell, boolean isTombstone, boolean conservative) {
+        WriteInfo write = WriteInfo.of(conservative ? TABLE_REF : TABLE_REF2, cell, isTombstone, timestamp);
         writer.enqueue(ImmutableList.of(write));
-        return WriteInfoPartitioner.getShard(write);
+        return write.toShard(SHARDS);
     }
 
-    public static List<WriteInfo> writeToUniqueCellsInSameShard(KvsSweepQueueWriter writer, long timestamp, int number, boolean conservative) {
+    public static List<WriteInfo> writeToUniqueCellsInSameShard(KvsSweepQueueWriter writer, long timestamp, int number,
+            boolean conservative) {
         TableReference tableRef = conservative ? TABLE_REF : TABLE_REF2;
         List<WriteInfo> result = new ArrayList<>();
         for (int i = 0; i < number; i++) {
             Cell cell = getCellWithFixedHash(i);
-            result.add(WriteInfo.of(tableRef, cell, timestamp));
+            result.add(WriteInfo.of(tableRef, cell, true, timestamp));
```

Comment:
true is slightly ugly - maybe replace `of` with a couple of methods to hide the true/false?